core/templates/dev/head/css/oppia.css

```diff
@@ -3869,8 +3864,12 @@ md-card.preview-conversation-skin-supplemental-card {
   margin-right: 0;
 }
 
-.oppia-about-credits-letter-groups {
+.oppia-about-credits-letter-groups.three-col{
   margin: 0 0 10% 0;
+  -moz-column-count: 3;
+  -moz-column-gap: 20px;
+  -webkit-column-count: 3;
+  -webkit-column-gap: 20px;
```

Comment:
could you also add
```
column-count: 3;
column-gap: 20px;
```
for compatibility? 
(see https://developer.mozilla.org/en-US/docs/Web/CSS/column-count)